app/controllers/jobs_controller.rb

```diff
@@ -21,24 +36,40 @@ def create
   end
 
   def show
-    @job = Job.find(params[:id])
+    @companies = Company.find(params[:company_id])
+    @job       = Job.find(params[:id])
+    @job_order = Job.find(params[:id]).order_by_time_created
```

Comment:
you're performing the same query (`Job.find(params[:id]`) twice. this is unnecessary. the more db queries we have, the longer our app will take to load. we want to minimize these queries.